portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -30,6 +30,11 @@
   private static final String ENTITY_PROPERTY_KEY_5 = "time";
   private static final String ENTITY_PROPERTY_KEY_6 = "address";
   private static final String ENTITY_PROPERTY_KEY_7 = "description";
+  private static final Double LAT_NORTH = 31.747628;
```

Comment:
nit: LAT_NORTH_LIMIT is clearer
same change for the ones below

Reply:
Sounds good, thank you!